### DISCUSSION TITLE
Enabling mtls in RabbitMQ operator

### DISCUSSION THREAD

**Discussion by paywagh11** [rabbitmq/cluster-operator#1526] [Q&A]
I got a request to enable mtls in my 3 node RMQ cluster operator.
Earlier I was using self signed certificate for webhook in messaging topology operator for tls.

RMQ server's cert value is stored in secret and ca_cert value provided in validating webhook config and mounted the Volume in the same.
Could you please help me in understanding where to store the Client certificates (in secret?) and how RMQ server will be able to access those Client certs (through Volume mount in validating webhook config?)?

**Comment by Zerpet** [comment]
I'm unsure where you want to enable mTLS, because you are mentioning a 3-node RMQ cluster operator, and RMQ server. Note that those are two different components. RabbitMQ server is the data service that acts as a message broker, and it's what your applications connect to. This is the Pod with RabbitMQ that results as a consequence of deploying a `RabbitmqCluster` object in Kubernetes. 
On the other hand, there is the RabbitMQ Cluster Operator, usually named cluster operator, or just "the operator". This a component responsible for configuring and deploying RabbitMQ servers when a`RabbitmqCluster` object is deployed in Kubernetes. This usually does not run with more than 1 replica, although it is capable of running with any odd number.

If you want to enble mTLS in rabbitmq itself, you can follow this doc: https://rabbitmq.com/kubernetes/operator/using-operator.html#tls
